src/sdk.rs

```diff
@@ -262,6 +262,31 @@ pub fn remove_storage(key: &[u8]) {
     }
 }
 
+pub fn register_len(register_id: u64) -> Option<u64> {
+    let len = unsafe { exports::register_len(register_id) };
+
+    if len == u64::MAX {
```

Comment:
Notice this is interacting with the low level NEAR blockchain API. I took [this code from `near-sdk`](https://github.com/near/near-sdk-rs/blob/master/near-sdk/src/environment/env.rs#L135-L145) and adapted it.

Reply:
I think duplicating the comment from `nearcore` on `exports::register_len` could be helpful for future devs (so that we don't have to go back to the low-level documentation all the time)

```
    /// Returns the size of the blob stored in the given register.
    /// * If register is used, then returns the size, which can potentially be zero;
    /// * If register is not used, returns `u64::MAX`
```